### ISSUE TITLE
Updates for change to symbol `MARSHES1P`

### ISSUE THREAD

**Issue by DavidGrant-NIWC** [iho-ohi/S-101_Portrayal-Catalogue#458]
- [ ] Replace the content of MARSHES1P with the content of MARSHE01
- [ ] Update Vegetation.lua to reference `MARSHES1P` vice `MARSHE01`
- [ ] Remove MARSHE01 (symbol and catalog entry)

> Alvaro email of 10DEC2024:
>Working on the ‘Easy Guide S57->S101’ for the PsWG, I realised the Vegetation Rule categoryOfVegetation=11 (Reed) is pointing to symbol MARSHE01 that is not a valid symbol in the Gi Registry.
When I tried to register MARSHE01 (as a new symbol for Reed pint features), Jeff pointed out that it was identical to MARSHE1P (used by the Area pattern) and recommended registering the updated symbol a s a new version of MARSHES1P and not as a brand new symbol MAESHE01. So I did.
In short, the symbol to be used for Point Vegetation/categoryOfVegettaion=11 has to be MARSHES1P and not MARSHE01.

**Comment by HyacinthAbad-NIWC** [comment]
As of Jan 2026,
I verified that both MARSHES1P and MARSHE01 display the same symbol.

I will submit a pull request that completes the checkboxes Dave posted here ^^.
